test/unittests/commands/validate_data/fixtures/validation_result.py

```diff
@@ -19,8 +17,7 @@
 from test.unittests.commands.validate_data.constants import PREFIXES
 
 
-@cache
-@pytest.fixture
+@pytest.fixture(scope="module")
```

Comment:
A strange behaviour of the new pytest version cannot handle a `@cache` decorator on fixtures. But it's preferrable anyways to use pytest's own caching mechanism.